feat(content): In the FW Hope Recon 1C surrender case, make the Gunboat crew join the Oathkeepers (#6451)
endless-sky/endless-sky: data/human/free worlds start.txt

			`	The landing crew has absolutely nowhere to hide in this glacial field; you pick them off easily. There is no response from the ship, so you land beside it and prepare to board it. But, the moment you open your hatch, the other ship roars to life, powering up its engines and beating a hasty retreat while firing a few parting shots at you from its turret. You fire up your own engines and pursue it...`
				launch
			
			label accept
			apply
				karma ++
				set "fw accepted Susquehanna's word of honor"
			`	Most ships are equipped with external speakers for exactly this sort of situation. You target the ship, and your scanners identify it as the <npc>.`
			`	"Crew of the <npc>, this is Captain <last> of the Free Worlds ship <ship>," you announce. "I accept your surrender, but will you each give me your word of honor that you will take no further part in fighting the Free Worlds?"`

> 			`	The soldier appear to think for a brief moment before one of them holds their hand high in from of them, giving you a thumbs-up sign. The rest of the soldiers quickly follow suit without any hesitation.`

Reviewer: Oops, this should be plural.
```suggestion
			`	The soldiers appear to think for a brief moment before one of them holds their hand high in from of them, giving you a thumbs-up sign. The rest of the soldiers quickly follow suit without any hesitation.`
```

			`	"Thank you," you reply. "You may return to your ship and fly back to Republic space."`

> 			`	The Navy soldiers exchange looks, apparently surprised, but they recover quickly. One of them snaps to attention and salutes you. You return the salute, and then, realizing their captain can't see you, flash your ship's external lights in acknowledgment. The <npc>'s crew fall into formation, march back to their ship, fire up their engines, and take off.`
> 			`	Their weapon systems remain unarmed and inactive, as far as your sensors can tell.`

Reviewer: Collapse these two lines together.
```suggestion
			`	The Navy soldiers exchange looks, apparently surprised, but they recover quickly. One of them snaps to attention and salutes you. You return the salute, and then, realizing their captain can't see you, flash your ship's external lights in acknowledgment. The <npc>'s crew fall into formation, march back to their ship, fire up their engines, and take off. Their weapon systems remain unarmed and inactive, as far as your sensors can tell.`
```

			`	The landing crew has absolutely nowhere to hide in this glacial field; you pick them off easily. There is no response from the ship, so you land beside it and prepare to board it. But, the moment you open your hatch, the other ship roars to life, powering up its engines and beating a hasty retreat while firing a few parting shots at you from its turret. You fire up your own engines and pursue it...`
				launch
			
			label accept
			apply
				karma ++
				set "fw accepted Susquehanna's word of honor"

> 			`	Most ships are equipped with external speakers for exactly this sort of situation.`

Reviewer: ```suggestion
			`	Most ships are equipped with external speakers for exactly this sort of situation. You target the ship, and your scanners identify it as the <npc>.`
```

> 			`	"Crew of the R.N.S. Susquehanna, this is Captain <last> of the Free Worlds ship <ship>," you announce. "Will you each give me your word of honor that you will take no further part in fighting the Free Worlds?"`

Reviewer: > Let the player learn the Gunboat's name via whatever mechanism lets the player learn names of ships while in flight, even if they're beyond radar range

Perhaps just say something to the effect of, "You target the Gunboat and your ship's scanners identify it as the R.N.S. Susquehanna."

> 			`	"Crew of the R.N.S. Susquehanna, this is Captain <last> of the Free Worlds ship <ship>," you announce. "Will you each give me your word of honor that you will take no further part in fighting the Free Worlds?"`

Reviewer: ```suggestion
			`	"Crew of the <npc>, this is Captain <last> of the Free Worlds ship <ship>," you announce. "I accept your surrender, but will you each give me your word of honor that you will take no further part in fighting the Free Worlds?"`
```

> 			`	The Gunboat captain thinks for a brief moment and gives you a thumbs-up sign, their hand held high in the air. Their crew follows suit without any visible hesitation.`

Reviewer: It's described before that the crew raises their hands in surrender, so their hands are already held high in the air, making it weird to me to describe again that their hands our up. And as with the name of the ship, there's no information prior to this point highlighting a captain among the crew on the ground. We also know that there are still crew on the ship, and I imagine the captain would be one of those still on the ship as opposed to being on the ground.

Given that this mission makes it known that most ships have external speakers, I'd imagine Navy ships also have such speakers, and since the captain of the Gunboat is likely still on the ship, perhaps the captain uses the Gunboat's speakers to respond as opposed to relying on the hand gestures of those on the ground.

Reviewer: > No, we don't know that.

We as in we the content creators, not we the players at this point in the mission.

> If the Gunboat is crewed, there's no need for speakers at all

I don't see how Navy ships being able to use radio to talk to each other precludes them from being able to have external speakers for those situations where they need to talk to someone outside the ship who doesn't have a radio. It's not as if the player's ship has speakers because it doesn't have a radio.

Author: > We as in we the content creators, not we the players at this point in the mission.

Even as a content creator I don't think we "know" that there's someone on board the Gunboat. We could make it so, but I don't think it's already so.

> I don't see how Navy ships being able to use radio to talk to each other precludes them from being able to have external speakers for those situations where they need to talk to someone outside the ship who doesn't have a radio. It's not as if the player's ship has speakers because it doesn't have a radio.

Sure. Of course the Gunboat _has_ speakers. It's just that if there's someone on board the Gunboat, then the Gunboat would communicate with the player by radio, not by speakers.

> 			`	The Gunboat captain thinks for a brief moment and gives you a thumbs-up sign, their hand held high in the air. Their crew follows suit without any visible hesitation.`

Reviewer: ```suggestion
			`	The soldier appear to think for a brief moment before one of them holds their hand high in from of them, giving you a thumbs-up sign. The rest of the soldiers quickly follow suit without any hesitation.`
```

> 			`	You consider the situation and decide not to bother with landing and taking your new prisoners on board.`

Reviewer: This line seems unnecessary to me. If they've given their word not to fight, why would you need to think about landing to take prisoners?

Author: > This line seems unnecessary to me.

Hmm, yes. It's backward, isn't it? This sentence should come before offering the parole deal to the Navy soldiers, not after.

> If they've given their word not to fight, why would you need to think about landing to take prisoners?

Because the player might not be willing to trust a random Navy captain's word enough to let them take off in a fully armed Gunboat, at this point? In the Kornephoros prisoners exchange, the Navy soldiers were returned to Republic space in FW ships, and the cease fire enabling this was negotiated by Alondo with... it doesn't say with whom, but I guess it would have been the Navy brass or the Republic government, not just a random captain.

> 			`	You consider the situation and decide not to bother with landing and taking your new prisoners on board.`

Reviewer: ```suggestion
```

			`	"Thank you," you reply. "You may return to your ship and fly back to Republic space."`

> 			`	The Navy sailors exchange looks, apparently surprised, but they recover quickly. Their captain snaps to attention and salutes you. You return the salute, and then, realizing their captain can't see you, flash your ship's external lights in acknowledgment. The R.N.S. Susquehanna's crew fall into formation, march back to their ship, fire up their engines, and take off.`

Reviewer: "Sailor" has never been used to describe members of the Navy. Typically they're referred to as Navy soldiers.

This interaction seems pretty awkward to me and could probably be reworked, but should we go the route of the Gunboat responding with its speakers this would get scrapped anyway.

Author: > "Sailor" has never been used to describe members of the Navy. Typically they're referred to as Navy soldiers.

OK, I'll change it to "soldier".

> This interaction seems pretty awkward to me and could probably be reworked, but should we go the route of the Gunboat responding with its speakers this would get scrapped anyway.

Could you be more specific about what's awkward?

> 			`	The Navy sailors exchange looks, apparently surprised, but they recover quickly. Their captain snaps to attention and salutes you. You return the salute, and then, realizing their captain can't see you, flash your ship's external lights in acknowledgment. The R.N.S. Susquehanna's crew fall into formation, march back to their ship, fire up their engines, and take off.`

Reviewer: ```suggestion
			`	The Navy soldiers exchange looks, apparently surprised, but they recover quickly. One of them snaps to attention and salutes you. You return the salute, and then, realizing their captain can't see you, flash your ship's external lights in acknowledgment. The <npc>'s crew fall into formation, march back to their ship, fire up their engines, and take off.`
```

			`	Their weapon systems remain unarmed and inactive, as far as your sensors can tell.`

> 			`	Your mission on this planet done, you launch into space too, monitoring the Navy ship. They gave you their word of honor that they would leave this system...`

Reviewer: This last bit seems to be a carryover from the original paragraph but no longer seems necessary and is more redundant than anything.
```suggestion
			`	Your mission on this planet done, you launch into space too, monitoring the Navy ship.`
```

Author: Let's revisit this after we agree on the other points.

> 			`	Your mission on this planet done, you launch into space too, monitoring the Navy ship. They gave you their word of honor that they would leave this system...`

Reviewer: I think the suggestion on this line from the other review should still be done.
